rest/handler.go

```diff
@@ -783,6 +800,19 @@ func (h *handler) writeJSON(value interface{}) {
 	h.writeJSONStatus(http.StatusOK, value)
 }
 
+func (h *handler) writeJavascript(js string) {
+	if !h.requestAccepts("text/plain") {
```

Comment:
We're checking for text/plain, but then sending as text/javascript.  Is that intentional?  

Also same question as above about application/javascript instead of text/javascript.

Reply:
Oops, copy-paste error. I've pulled this stuff out into a `writeWithMimetypeStatus` to avoid the duplication of mime types.